lib/absinthe/type/built_ins/scalars.ex

```diff
@@ -59,6 +59,48 @@ defmodule Absinthe.Type.BuiltIns.Scalars do
     parse parse_with([Absinthe.Blueprint.Input.Boolean], &parse_boolean/1)
   end
 
+  scalar :utc_datetime, name: "DateTime" do
```

Comment:
Yeah given that we're using the native Elixir date time handling I think we should go with `:datetime` as always UTC.